ui/StatusQ/src/StatusQ/Components/StatusChatList.qml

```diff
@@ -39,248 +39,238 @@ Item {
-                        highlightWhenCreated: !!model.highlight
-                        selected: (model.active && root.highlightItem)
-                        asset.emoji: !!model.emoji ? model.emoji : ""
-                        asset.color: !!model.color ? model.color : Theme.palette.userCustomizationColors[model.colorId]
-                        asset.isImage: model.icon.includes("data")
-                        asset.name: model.icon
-                        ringSettings.ringSpecModel: type === StatusChatListItem.Type.OneToOneChat && root.isEnsVerified(chatId) ? undefined : model.colorHash
-                        onlineStatus: !!model.onlineStatus ? model.onlineStatus : StatusChatListItem.OnlineStatus.Inactive
-
-                        sensor.cursorShape: dragSensor.cursorShape
-
-                        onClicked: {
-                            highlightWhenCreated = false
-
-                            if (mouse.button === Qt.RightButton && !!root.popupMenu) {
-                                statusChatListItem.highlighted = true
-
-                                let originalOpenHandler = popupMenuSlot.item.openHandler
-                                let originalCloseHandler = popupMenuSlot.item.closeHandler
-
-                                popupMenuSlot.item.openHandler = function () {
-                                    if (!!originalOpenHandler) {
-                                        originalOpenHandler(statusChatListItem.chatId)
+                        onMouseXChanged: {
+                            if ((Math.abs(startX - mouseX) > 1) && pressed) {
+                                active = true
+                            }
+                        }
+                        onActiveChanged: d.destinationPosition = -1
+
+                        StatusChatListItem {
+                            id: statusChatListItem
+
+                            width: parent.width
+                            opacity: dragSensor.active ? 0.0 : 1.0
+                            originalOrder: model.position
+                            chatId: model.itemId
+                            categoryId: model.categoryId
+                            name: model.name
+                            type: !!model.type ? model.type : StatusChatListItem.Type.CommunityChat
+                            muted: model.muted
+                            hasUnreadMessages: model.hasUnreadMessages
+                            notificationsCount: model.notificationsCount
+                            highlightWhenCreated: !!model.highlight
+                            selected: (model.active && root.highlightItem)
+                            asset.emoji: !!model.emoji ? model.emoji : ""
+                            asset.color: !!model.color ? model.color : Theme.palette.userCustomizationColors[model.colorId]
+                            asset.isImage: model.icon.includes("data")
+                            asset.name: model.icon
```

Comment:
Is it currently possible to have images defined for channels?

Reply:
Yes. For 1-1 channels. It still works. I guess the image is inside `icon`